.changeset/clever-timers-wait.md

```diff
@@ -0,0 +1,6 @@
+---
+"@kaizen/design-tokens": minor
+"@kaizen/components": minor
+---
+
+Update docs
```

Comment:
This can be empty, you don't need to version anything for docs to be published. (will try cover this in the guide too)
`yarn changeset --empty`